modules/mcc/src/checker_model.cpp

```diff
@@ -420,7 +420,7 @@ void CCheckerImpl::setPatchCenters(std::vector<Point2f> _patchCenters)
 }
 void CCheckerImpl::setPatchBoundingBoxes(std::vector<Point2f> _patchBoundingBoxes)
 {
-    //Use of this function is not recommended as it can cause contradictory information in patBoundingBoxes and bo
+    //Use of this function is not recommended as it can cause contradictory information in patBoundingBoxes and box
```

Comment:
I don't really know the context, but is this function then only meant to be used internally. Why allow a function that can make the state fo the program contradictory? Make it private instead?

Reply:
Currently, all the geometric details about the chart are calculated entirely from the position of corners (calculate() function does this). But sometimes giving the users the ability to correct some of the details might be better. Like if the user does not want to extract the colour from the centre but from some corner part or maybe if some prediction is wrong, they can change the location of bounding boxes to fit their need. Also perhaps in future, finding corners using centre positions of each patch can be added, not sure if it will be useful though.